bin/helpers/utils.js

```diff
@@ -131,3 +131,8 @@ exports.exportResults = (buildId, buildUrl) => {
     }
   });
 }
+
+exports.deleteResults = () => {
+  fs.unlink("log/build_results.txt", function (err){
```

Comment:
add log line for error?

Reply:
The unlink function throws an error even if there is no file present in the directory. And it is a validate case. Hence haven't logged any error.